test/dialogmote/DialogmoteInnkallingSkjema/DialogmoteInnkallingSkjemaTest.tsx

```diff
@@ -291,6 +291,12 @@ describe("DialogmoteInnkallingSkjema", () => {
     expect(
       screen.getByText(getInnkallingTexts(Malform.NYNORSK).arbeidstaker.intro2)
     ).to.exist;
+
+    // Cleanup global state for other tests
+    const malformRadioBokmal = screen.getByRole("radio", {
+      name: "Bokmål",
+    });
+    userEvent.click(malformRadioBokmal);
```

Comment:
Ettersom vi i denne testen slår på nynorsk dialogmøteinnkalling, så fortsetter den globale staten å være nynorsk for resten av testene. Vi har derimot lagt opp til bokmål-tekster i testene.

Reply:
Det er kanskje teit å "fikse" globalstate ved å trykke på skjemaelement. Er det ikke bedre å bare endre globalstate i så fall? 🤔

Reply:
Mulig, jeg kan undersøke om det er enkelt å bare ta inn hooken i testen og oppdatere i stedet 👍🏼

Reply:
Går an å gjøre dette etter beforeEach øverst i testen
```
afterEach(() => {
    localStorage.setItem(StoreKey.MALFORM, Malform.BOKMAL);
  });
```